module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>use cases (needs a space)</p><p> </p><p>I would just say "see the HOOMD-blue <a href="https://hoomd-blue.readthedocs.io/en/latest/index.html" target="_blank">documentation</a> and <a href="https://github.com/glotzerlab/hoomd-examples" target="_blank">examples</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>